README.md

```diff
@@ -6,7 +6,7 @@ _[Contentful CMS][2] + [express][3] + [styled components][4]_
 
 1.  Clone the repo locally `git clone https://github.com/state-matters/state-matters-website.git`
 1.  Install server & client deps `npm install && cd client && yarn`
```

Comment:
`yarn` isn't actually used in our app. It's possible to use but no longer to the server and client have different package.json files.